pkg/splash/splash.go

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package splash contains boot splash images.
+package splash
+
+import (
+	_ "embed"
+)
+
+//go:embed sidero.bmp
+var sideroBMP []byte
+
+// GetBoot returns boot splash image.
+func GetBoot() []byte {
```

Comment:
Get
```suggestion
func GetSplash() []byte {
```

Reply:
`splash.GetSplash()` sounds weird, probably get `splash.GetEarlyBootBMP()` or something like that